bake/remote.go

```diff
@@ -6,8 +6,8 @@ import (
 	"context"
 	"strings"
 
-	"github.com/docker/buildx/build"
 	"github.com/docker/buildx/driver"
+	"github.com/docker/buildx/driver/driverutil"
```

Comment:
The types in this pkg should just go to `driver` pkg. I think the functions fit the `storeutil` pkg. Can be in a separate file there.